docs/source/index.md

```diff
@@ -1,9 +1,45 @@
-# Jupyter Project Documentation
+# Project Jupyter Documentation
 
-**Welcome to the Jupyter Project documentation**. This website acts as "meta" documentation for the Jupyter ecosystem.
-It has a collection of resources to navigate the tools and communities in this ecosystem, and to help you get started.
+Welcome to the Project Jupyter documentation site. As you'll read below,
+Jupyter is a large umbrella project that covers many different software
+offerings and tools, including the popular **JuyterLab** and **Jupyter**
+**Notebook** web-based notebook authoring and editing applications. This
+site acts as (meta) documentation for the entire Jupyter project.
 
-## Start Here
+## What is Jupyter?
+
+The name "Jupyter" is an ambiguous, overloaded term that can mean several
```

Comment:
Welcome to the Project Jupyter documentation site. Jupyter is the
umbrella project that covers many different software offerings and
tools, including the popular **Notebook**, **Jupyter**, and
**JuyterLab** web-based notebook authoring and editing applications. 
This site acts as (meta) documentation for the entire Jupyter project.

The major subprojects from Jupyter are:

Reply:
My proposed wording states what Jiupyter is and makes it unambiguous.

I would order the major components as notebook, Jupyter, JupyterLab because that goes from simple to complex.

Reply:
> Welcome to the Project Jupyter documentation site. Jupyter is the umbrella project that covers many different software offerings and tools, including the popular **Notebook**, **Jupyter**, and **JuyterLab** web-based notebook authoring and editing applications. This site acts as (meta) documentation for the entire Jupyter project.
> 
> The major subprojects from Jupyter are:

This is good. I think we can remove the second of those three items and make it Notebook and Lab (second item, Jupyter, is just the umbrella project). Will update.

Reply:
I think we need to address naming ambiguity somehow, because the current reality is that a lot of confusion stems from EXISTING multiple/ambiguous usages that is VERY, very widespread. Not sure the best way/place to acknowledge this so that people aren't caught off guard and then end up more confused...there's a deeper issue here about bad naming choices, but in the short term I'd like some kind of solution that addresses present realities but also takes a stance on the best language to use going forward...

Reply:
I agree we need to address the ambiguity... but not in the opening paragraph of this documentation.  We want the initial statements to be coherent and forward looking, not cluttered with an apology for past ambiguity.  Maybe as a point immediately after the coherent statements, under the title of explaining past ambiguity.

Reply:
What are the main pieces of ambiguity:

To my knowledge Jupyter refers to
1. The Jupyter notebook
1. The Jupyter project - with an executive council... The Jupyter project was responsible for putting on JupyterCon
1. The command used to start `jupyter notebook` or `jupyter lab`
1. The Jupyter of "Jupyter Widgets"

What else am I missing?

Reply:
Can you clarify, where that would be put? I'm trying to figure out what to do with that "What is Jupyter" section now that we have additional clarifications in the first paragraph (much of it can probably be eliminated now?).